spec/features/admin/admins_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Edit admin`s password page', js: true do
+  let!(:admin){ create(:admin, email: 'test1@gmail.com', password: '12345admin') }
```

Comment:
why do we need set `email` and `password` here. Doesn't Factory create it?

Reply:
Fixed that. In last commit used only Factory